docs/user/agent/mce-integration.md

```diff
@@ -3,6 +3,106 @@
 This document describes how to install the multicluster engine for Kubernetes operator (mce) and how to deploy the cluster zero (hub) using the agent-based installer for Openshift.
 The procedure is partially automated, and it will require some manual steps after the initial cluster deployment.
 
+## Installing while disconnected from the greater internet
+
+### Environment Pre-requisites
+
+[Create a local image registry in your environment.](https://docs.openshift.com/container-platform/4.11/installing/disconnected_install/index.html) Make note of the DNS name and port of your local registry.
+
+Download the [oc mirror](https://github.com/openshift/oc-mirror) tool.
+
+#### Agent Installer pre-requisites
+
+For the complete pre-requisites description, please check section 1.2.1 in [[1]](#1).
+
+Create a folder (`<asset dir>`) and place a valid `install-config.yaml` and `agent-config.yaml`.
+
+### Mirror the Openshift release and the MCE operator
+
+Using the oc mirror tool mirror the desired Openshift release (4.11+). Here is an example ImageSetConfiguration for OCP 4.11:
+
+__ocp-mce-imageset.yaml__
+```
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v1alpha2
+archiveSize: 4
+storageConfig:
+  imageURL: <your-local-registry-dns-name>:<your-local-registry-port>/mirror/oc-mirror-metadata
+  skipTLS: true
+mirror:
+  platform:
+    architectures:
+      - "amd64"
+    channels:
+      - name: stable-4.11
+        type: ocp
+  additionalImages:
+    - name: registry.redhat.io/ubi8/ubi:latest
+  operators:
+    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.11
+      packages:
+        - name: multicluster-engine
+        - name: local-storage-operator
+```
+
+With this file you will be able to use this command to mirror the OCP release and the MCE and LSO operators:
```

Comment:
nit: MCE is defined above, but this is the first use of LSO.

Reply:
Ah.. right there was another sentence that got edited out. I'll put it in the first sentence of this section